11ty/documentation/development.md

```diff
@@ -46,3 +46,7 @@ npm run build
 ```
 
 This will generate a minified CSS bundle as well as run Eleventy’s production build. Build files are located in `dist`.
+
+## Contributing
+
+Follow the [contributing guidelines](https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md) to start contributing.
```

Comment:
I’d like to have this section further at the top. Maybe instead of a section, we can appnd a sentence to the intro.

> We’ve compiled a list of contribution guidelines to make it easy for you to start work on the project.

Something like this, maybe?

Reply:
I like this.

Reply:
Thank you for the feedback. Will make this change!